src/ziggurat/producer.clj

```diff
@@ -57,7 +58,7 @@
    :methods  [^{:static true} [send [String String Object Object] java.util.concurrent.Future]
               ^{:static true} [send [String String int Object Object] java.util.concurrent.Future]]))
 
-(defn *implements-serializer?* [serializer-class]
+(defn ^:dynamic *implements-serializer?* [serializer-class]
```

Comment:
Why do we need the dynamic tag? I think we are not using this as dynamic variable anywhere.

Probably, we can get rid of those leading and following '*'

Reply:
is `implements-serializer` gets rebinded from somewhere else?